crm/controllers/user.controller.js

```diff
@@ -46,4 +47,13 @@ exports.findAllUsers = async (req, res) => {
  * As a ADMIN user, I should be able to update the status of the user
  * 
  * Time till 11:25 PM 
- */
\ No newline at end of file
+ */
+exports.updateAnUser = async (req, res) => {
+    try {
+        const user = await User.findOneAndUpdate({userId : req.userId}, {$set : {userStatus : "APPROVED"}});
```

Comment:
req.userId will be of ADMIN ...

Engineer userId has to be read from the path param